scalahost/nsc/src/main/scala/scala/meta/internal/scalahost/ScalahostPipeline.scala

```diff
@@ -34,7 +34,10 @@ trait ScalahostPipeline extends DatabaseOps { self: ScalahostPlugin =>
         global.settings.outputDirs.getSingleOutput
           .map(_.file.getAbsolutePath)
           .getOrElse(global.settings.d.value))
-      val assumedSourcepath = Sourcepath(".")
+      // NOTE: user.dir is not a great default for sourcepaths since it will include
+      // irrelevant directories such as target. This parameter should ideall be passed by
+      // the build integration as a compiler flag -Xplugin:scalahost:xxx.
+      val assumedSourcepath = Sourcepath(sys.props("user.dir"))
```

Comment:
One is AbsolutePath and the other is RelativePath. `.` can evaluate to different paths at different times, which is not fun to debug. Sourcepath should IMO accept a `Seq[AbsolutePath]` instead of `String`. 

I didn't dig exactly into why `.` failed in the scripted tests, but I suspect it was related to how scripted tests are run from a temporary directory.